### PR TITLE
added projects controller and route and fixed module/Projects

### PR DESCRIPTION
**Pull request by AmalZubidat**

### CODE REVIEW

**Comment by Karyum** [comment]
Just adding this comment for clarification, this controller is meant for the dropdown list on the homepage

**Comment by AmalZubidat** [comment]
Resolved Commits

File: controllers/projects.controller.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const catchAsync = require('../utils/catchAsync')
+const ApiError = require('../utils/ApiError')
+const db = require('../database/connection')
```
Comment (AmalZubidat) [inline controllers/projects.controller.js]: db module shall not be used inside controller use models/project. 
note d is not used.

File: controllers/projects.controller.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const { getProjects } = require('../database/models/Project')
+const getProjectsController = catchAsync(async (req, res) => {
+  const data = await getProjects()
+  if (data) {
+    res.status(200).send({
+      message: 'all projects retrieved successfully.',
+      data: data
+    })
+  }
+})
```
Comment (AmalZubidat) [inline controllers/projects.controller.js:17]: if not !data it should return 200 or something. 
instead of send use json.

Comment (AmalZubidat) [reply controllers/projects.controller.js:17]: respose 200 and message no projects found,

**Review by AmalZubidat — COMMENTED** [review]
please make the required changes

**Review by tawfiknasser — APPROVED** [review]

**Review by joumanakh — APPROVED** [review]

**Pull request merged**